Makefile

```diff
@@ -488,6 +488,12 @@ endif
 
 include lib/stack_protector/stack_protector.mk
 
+ifeq (${MULTI_CONSOLE_API},1)
+BL_COMMON_SOURCES	+=	drivers/console/${ARCH}/multi_console.S
+else
+BL_COMMON_SOURCES	+=	drivers/console/${ARCH}/deprecated_console.S
```

Comment:
Including the `deprecated_console` by default is causing build failure on rcar platform:

./build/rcar/debug/bl2/scif.o:/work/workspace/workspace/tf-worker@9/trusted_firmware/drivers/renesas/rcar/scif/scif.S:128: first defined here
11:45:09 ./build/rcar/debug/bl2/deprecated_console.o: In function `console_uninit':
11:45:09 (.text.asm.console_uninit+0x0): multiple definition of `console_uninit'
11:45:09 ./build/rcar/debug/bl2/scif.o:/work/workspace/workspace/tf-worker@9/trusted_firmware/drivers/renesas/rcar/scif/scif.S:128: first defined here

Since the `deprecated_console` is set to be removed soon, better not to bother with the change.